OrbitGl/TrackAccessibility.cpp

```diff
@@ -0,0 +1,120 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "TrackAccessibility.h"
+
+#include "GlCanvas.h"
+#include "TimeGraph.h"
+#include "Track.h"
+
+namespace orbit_gl {
+
+using orbit_accessibility::AccessibleInterface;
+
+const AccessibleInterface* AccessibleTrackContent::AccessibleParent() const {
+  return track_->AccessibilityInterface();
+}
+
+std::string AccessibleTrackContent::AccessibleName() const {
+  CHECK(track_ != nullptr);
+  return track_->GetName() + "_content";
+}
+
+AccessibilityRect AccessibleTrackContent::AccessibleLocalRect() const {
+  CHECK(track_ != nullptr);
+  CHECK(track_->GetTimeGraph() != nullptr);
+
+  auto& layout = track_->GetTimeGraph()->GetLayout();
+  return AccessibilityRect(0, static_cast<int>(layout.GetTrackTabHeight()),
+                           static_cast<int>(track_->GetSize()[0]),
+                           static_cast<int>(track_->GetSize()[1]));
+}
```

Comment:
I would prefer querying the Track for its position rather than going through the layout. It would be easier to support tracks that wouldn't have a tab for example. It seems to me that this logic should centralized in the tracks themselves. Also, shouldn't the rectangle include the tab?

Reply:
Overall I agree, but I'd rather do that in a subsequent PR that tackles the problem a bit more generic. The problem here is that I'm splitting the Track into 3 UI components (container, tab, and content), but the Track itself doesn't know about this. Making this work without layouting information would require adding a bunch of methods to the track (e.g. `GetTabPos`, `GetTabSize`, `GetContentPos`, `GetContentSize`).

On top of that, the Tracks are IMO not really consistent in what they consider "position" and "size", because both of them do not take the Tabs into account, so I'd either need to change how `Track::GetSize` and `Track::GetPos` work, or add additional methods.

A better solution would IMO be introducing a general interface for UI-Controls that exposes pos / size and parent / child relationsships (maybe tied to the `Pickable` interface?), then the Tracks could actually create sub-controls for the Tabs and the Content and own the layout.

Reply:
Ok sounds good, thanks for the precision.

Reply:
> I'm ok if you change that and I can easily modify TrackManager (I think that is 1-line modification) to draw everything properly.

I'm shying away from this change a bit since I'm not sure which parts directly or indirectly rely on this convention atm. If it's ok with you, I would suggest to merge it as-is, I need to take care of rebasing on your changes anyways, and we could then tackle this afterwards.